app/models/barcode.rb

```diff
@@ -97,6 +98,7 @@ class Barcode < ApplicationRecord
     brants_bridge_v2
     uk_biocentre_v7
     east_london_genes_and_health
+    lemington_spa_v2
```

Comment:
I think it's spelled `leamington` rather than `lemington`.  https://www.warwickdc.gov.uk/info/20114/health_and_medical_care/1646/rosalind_franklin_laboratory_in_royal_leamington_spa

There are a few mentions in this pull request.

Reply:
I took the spelling from the issue. I will change it.

Reply:
Sure thing.  I did see that's what had happened after I posted my review 😄 .  I only know it's spelled differently because I used to live in the next town across.